src/index.html

```diff
@@ -5,8 +5,280 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
+  <header class="header">
+    <div class="logo">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo.png">
+      </a>
+    </div>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a class="nav__link nav__link--active" href="#apple">apple</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#samsung">samsung</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#phone">смартфоны и телефоны</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" data-qa ="nav-hover" href="#comp">ноутбуки и компьютеры</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#gadj">гаджеты</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#pad">планшеты</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#photo">фото</a>
+        </li>
+        <li class="nav__item nav__item--last">
+          <a class="nav__link" href="#video">видео</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
   <body>
-    <h1>Catalog</h1>
+    <main class="maincon">
+      <article class="cards maincon__product-card" data-qa ="card">
+        <div class="cards__container">
+          <div class="cards__photo">
+            <img src="images/imac.jpeg" alt="imac" class="cards__photo_product">
+          </div>
+          <h3 class="cards__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h3>
+
+          <div class="cards__code">
+            <p>Код товара: 195434</p>
+          </div>
```

Comment:
Don't use additional div
```suggestion
          <p class="cards__code">
            Код товара: 195434
          </p>
```